types/auth0-js/index.d.ts

```diff
@@ -270,10 +271,40 @@ export class WebAuth {
     signupAndAuthorize(options: any, callback: Auth0Callback<any>): void;
 
     /**
-     * Redirects to the auth0 logout page
+     * Logs in the user with username and password using the cross origin authentication (/co/authenticate) flow. You can use either `username` or `email` to identify the user, but `username` will take precedence over `email`.
+     * This only works when 3rd party cookies are enabled in the browser. After the /co/authenticate call, you'll have to use the {@link parseHash} function at the `redirectUri` specified in the constructor.
+     *
+     * @method login
+     * @param {Object} options options used in the {@link authorize} call after the login_ticket is acquired
+     * @param {String} [options.username] Username (mutually exclusive with email)
```

Comment:
This documentation should go in a type definition instead of in JSDoc. I.e.:
```ts
interface Login Options {
    /** Mutually exclusive with email. */
    username: string;
    ...
}
```